_components/in-page-navigation/accessibility-tests.md

```diff
@@ -0,0 +1,11 @@
+---
+permalink: /components/in-page-navigation/accessibility-tests/
+layout: accessibility-test
+component:
+ name: in page navigation
+title: In page navigation accessibility tests
+category: Components
+lead: Any USWDS in page navigation component should pass these manual accessibility tests.
+changelog:
+ key: 'component-in-page-navigation-accessibility'
```

Comment:
Again, two indents here.

```suggestion
  key: 'component-in-page-navigation-accessibility'
```

Reply:
Resolved in [5b4a912](https://github.com/uswds/uswds-site/pull/2863/commits/5b4a9120e93079ea8d1cc7b6e1c4a847bd6afb73)